app/docs/src/reference/actions.md

```diff
@@ -109,6 +110,67 @@ Refresh actions and qualifications are the only functions that can run directly
 on HEAD without going through a change set, since they don't modify resources -
 they only read current state or validate configuration.
 
+## Action Arguments
+
+Action functions take an `Input` argument. It has a `properties` field which
+contains an object that has:
+
+- The `si` properties
+
+  These are the core properties set as meta-data for the function. Name, color,
+  etc.
+
+- The `domain` properties
+
+  These are the properties specified in the schema itself.
```

Comment:
I wonder if we should mention that this is usually 1:1 with the cloud provider spec